src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -24,6 +25,8 @@ def process_result(message: Message[KafkaPayload | FilteredPayload]):
 
     try:
         result: CheckResult = UPTIME_RESULTS_CODEC.decode(message.payload.value)
+        if result["status"] == CHECKSTATUS_FAILURE:
+            create_issue_platform_occurrence(result)
 
         # XXX(epurkhiser): This consumer literally does nothing except log right now
```

Comment:
Can prob drop this comment. not critical if nothing else in the PR needs changed